libr/cons/dietline.c

```diff
@@ -442,7 +442,112 @@ R_API int r_line_hist_chop(const char *file, int limit) {
 	return 0;
 }
 
+static void draw_selection_widget () {
+	RCons *cons = r_cons_singleton ();
+	RSelWidget *sel_widget = I.sel_widget;
+	int y, pos_y = cons->rows, pos_x = r_str_ansi_len (I.prompt);
+
+	for (y = 0; y < sel_widget->options_len; y++) {
+		sel_widget->w = R_MAX (sel_widget->w, strlen (sel_widget->options[y]));
+	}
+	sel_widget->w = R_MIN (sel_widget->w, R_SELWIDGET_MAXW);
+
+	char *background_color = cons->color ? "\x1b[40m" : Color_INVERT; // XXX colors from palette
+	char *selected_color = cons->color ? "\x1b[41m" : Color_INVERT_RESET; // XXX colors from palette
+	bool scrollbar = sel_widget->options_len > R_SELWIDGET_MAXH;
+	int scrollbar_y = scrollbar
```

Comment:
I think it would be clearer to just do:
```
int scrollbar_y = 0, scrollbar_l = 0;
if (scrollbar) {
   scrollbar_y = ...;
   scrollbar_l = ...;
}